lib/components/AlbumScreen/speed_menu.dart

```diff
@@ -78,7 +77,7 @@ class _SpeedSliderState extends State<SpeedSlider> {
               setState(() {
                 _dragValue = value;
               });
-              FeedbackHelper.feedback(FeedbackType.impact);
+              FeedbackHelper.feedback(FeedbackType.heavy);
```

Comment:
I'll have to test how this feels, but it should definitely only be a short feedback